apps/academy/src/feature/link/ui/FAQToggleButton.tsx

```diff
@@ -3,16 +3,14 @@ import Image from 'next/image'
 import ArrowWebp from '/public/webp/arrow-down.webp'
 
 type Props = {
-  onClick: () => void
   isToggle: boolean
 }
 
-export function FAQToggleButton({ onClick, isToggle }: Props) {
+export function FAQToggleButton({ isToggle }: Props) {
   return (
-    <button
-      onClick={onClick}
-      className={`transform transition-transform duration-200 ${isToggle ? 'rotate-180' : ''}`}>
+    <button className={`transform duration-200 ${isToggle ? 'rotate-180' : ''}`}>
```

Comment:
```suggestion
    <button className={cn('transform duration-200', isToggle && 'rotate-180'}>
```

해당부분 어떠신지 모르겠는데 전 아래와 같이 하는게 좀 더 가독성이 좋아보이더라구요!

Reply:
[[MOBI-33-landing-page-bottom-mobile-publishing] Refactor: SubjectProps 타입 확장 수정, classname 조건문 수정](https://github.com/team-mobi/mobi-client/pull/14/commits/72d196230fef81f0369045e698e750e8b1b4be69)

빈값이니 &&가 좋겠네요. 해당 부분 반영하였습니다!